src/cache/cache.rs

```diff
@@ -373,30 +382,31 @@ impl Storage for opendal::Operator {
 }
 
 /// Normalize key `abcdef` into `a/b/c/abcdef`
+#[allow(dead_code)]
```

Comment:
Could it be removed altogether?

Reply:
This function will be used if feature like `s3`, `azure` enabled. Adding a `dead_code` is the most simple way to address clippy errors under `--no-default-feature`.